readme.txt

```diff
@@ -98,10 +98,20 @@ Please note that our support for the checkout block is still experimental and th
 
 == Changelog ==
 
-= 2.8.4 - 2021-08-17 =
+= 2.9.0 - 2021-xx-xx =
+* Add - Split discount line in timeline into variable fee and fixed fee.
+* Add - Order status validation for payments/orders/{order_id}/create_customer API.
+* Add - Add country code parameter to ExPlat API requests.
 * Fix - Align table items according to design correctly.
+* Fix - Fatal error if wcpay_multi_currency_enabled_currencies is a string.
+* Fix - Show the estimated deposit date in the transactions CSV export rather than the deposit ID.
+* Fix - Keep track of customer id in non logged in users.
+* Update - Bump minimum supported version of WooCommerce from 5.3 to 5.4.
+
+= 2.8.4 - 2021-08-17 =
 * Fix - Fix database connection error on account cache clear.
 * Fix - Fix fatal error logged when updating analytics data when account is not connected to Stripe.
+* Fix - Multi-Currency Compatibility fatal error with Subscriptions when account is not connected to Stripe.
```

Comment:
FYI, these changes are not reflected in changelog.txt

Reply:
I believe it's this entry? https://github.com/Automattic/woocommerce-payments/pull/2792/files#diff-b40cd67182487ef24807d9c9268329d35fbd96aa2b0a9cae69e2e0d746b1c666R16